### ISSUE TITLE
add dropout primitive

### ISSUE THREAD

**Issue by roddtalebi** [ezCGP/ezCGP#158]

**Comment by roddtalebi** [comment]
https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dropout

**rate** is the only arg i thing we should worry about. It takes in a float between (0,1) so maybe pick have the arg be one of [0.1, 0.2, 0.3, 0.4, 0.5] ? Don't see a need to go higher than that

Maybe can get away with using [this argtype](https://github.com/ezCGP/ezCGP/blob/2021S-BaseCodeDevelopment/codes/block_definitions/utilities/argument_types.py#L498)...it includes 0 and goes up to 0.95, but yolo

Or [this](https://github.com/ezCGP/ezCGP/blob/2021S-BaseCodeDevelopment/codes/block_definitions/utilities/argument_types.py#L459) which are ints from 1 to 10, inclusively. Then we can do ```arg/20``` to go from [0.1,0.5] in increments of 0.05
